src/protocols/rdp/guac_rdpdr/rdpdr_printer.c

```diff
@@ -165,9 +165,6 @@ static void guac_rdpdr_device_printer_iorequest_handler(guac_rdpdr_device* devic
 
 static void guac_rdpdr_device_printer_free_handler(guac_rdpdr_device* device) {
 
-    free(device->rdpdr);
-    free((char *) device->device_name);
```

Comment:
`device->device_name` *is* dynamically allocated in this case (printer) due to the changes in this PR. Only the filesystem one currently remains static and shouldn't be freed.

Reply:
Ah, yes.  Of course.

Reply:
So, actually, the `device->device_name` is passed by reference from the `settings->printer_name` pointer, so if I try to free both `device->device_name` (in this function) and `settings->printer_name` (in the rdp_settings.c free function), I get a double free or corruption error.  So I've removed it out of here.